extensions/sql-bindings/src/common/constants.ts

```diff
@@ -32,6 +32,8 @@ export const timeoutProjectError = localize('timeoutProjectError', 'Timed out wa
 export const errorNewAzureFunction = localize('errorNewAzureFunction', 'Error creating new Azure Function: {0}');
 export const azureFunctionsExtensionNotInstalled = localize('azureFunctionsExtensionNotInstalled', 'Azure Functions extension must be installed in order to use this feature.');
 export const azureFunctionsProjectMustBeOpened = localize('azureFunctionsProjectMustBeOpened', 'A C# Azure Functions project must be present in order to create a new Azure Function for this table.');
+export const chooseAConnection = localize('chooseAConnection', 'A SQL Server table must be selected to use this feature.');
```

Comment:
"A SQL table must be selected to use this feature."

(we support more than SQL Server so keeping it generic is good)

Reply:
Since I decided to add prompts I removed this since we can get all the required information to start the createAzureFunction.